app/views/content/life-as-a-teacher/explore-subjects/languages.md

```diff
@@ -3,6 +3,8 @@ title: Become a languages teacher
 title_paragraph: |-
   <p>
   Learning a language teaches pupils more than just grammar and vocabulary. As a languages teacher, you'll help pupils gain a greater appreciation and understanding of the world.</p>
+  <p>
+  Tax-free bursaries of $bursaries_postgraduate_languagesfrenchgermanspanish$ or scholarships of $scholarships_languagesfrenchgermanspanish$ are available for eligible trainee languages teachers.</p>
```

Comment:
This isn't correct @emmaachesongray - scholarships are only available for French, German and Spanish (if eligible). This will need rewording